util/revert.js

```diff
@@ -177,7 +196,7 @@ function cache(options, api) {
             const q = new Q(50);
 
             for (const feat of delta.features.features) {
-                q.defer(async (feat, done) => {
+                q.defer(async(feat, done) => {
```

Comment:
Why take out the space here?

Reply:
ESLint incorrectly doesn't like the space. I think it's parsing it as a named function. So easier to just remove the space then have to disable it for the line